scripts/pages/zoning/zoning.ts

```diff
@@ -1021,7 +1133,58 @@ class ZoneMenuHandler {
 			{
 				track: track,
 				segment: segment,
-				zone: newZone
+				zone: newZone,
+				region: newZone.regions[0]
+			},
+			true
+		);
+	}
+
+	addGlobalBhopRegion() {
+		if (!this.panels.globalBhop) throw new Error('Missing global bhop region panel!');
+		if (!this.mapZoneData.globalRegions) {
```

Comment:
Nit, don't need to block merging can resolve if you want, but can do 
```ts
			this.mapZoneData.globalRegions ??= {
				bhop: [],
				cancel: []
			};
```
Same thing in `addGlobalCancelRegion`